### DISCUSSION TITLE
Clarification of OffsetSpecification Timestamp usage

### DISCUSSION THREAD

**Discussion by willowglensystems** [rabbitmq/rabbitmq-stream-go-client#350] [Q&A]
Hello,

I had posted on the RabbitMQ google users group regarding the usage of the `OffsetSpecification.Timestamp()` API. See here: https://groups.google.com/g/rabbitmq-users/c/0xhLf1CVJ3Q

That post has lost of detail, so I won't repeat it here, but after one of the commenters mentioned the Streams message timestamp being in milliseconds, I updated my tests to use milliseconds and it now works as expected. I'm making this post just for posterity in case someone else has the same problem.

It would be great if comments could be added the `OffsetSpecifications` type and API, or an example of using the `Timestamp()` API could be added to the `examples/` directory. This usage was not obvious when reviewing this repo nor the RabbitMQ documentation.

Thanks!

**Comment by Gsantomaggio** [comment]
Thank you, @willowglensystems, for the details. 
Feel free to propose a PR with the example.
